lua/lazyvim/plugins/extras/ui/alpha.lua

```diff
@@ -23,11 +23,11 @@ return {
       dashboard.section.header.val = vim.split(logo, "\n")
       -- stylua: ignore
       dashboard.section.buttons.val = {
-        dashboard.button("f", " " .. " Find file",       "<cmd> Telescope find_files <cr>"),
-        dashboard.button("n", " " .. " New file",        "<cmd> ene <BAR> startinsert <cr>"),
-        dashboard.button("r", " " .. " Recent files",    "<cmd> Telescope oldfiles <cr>"),
-        dashboard.button("g", " " .. " Find text",       "<cmd> Telescope live_grep <cr>"),
-        dashboard.button("c", " " .. " Config",          "<cmd> lua require('lazyvim.util').telescope.config_files()() <cr>"),
+        dashboard.button("f", " " .. " Find file",       [[<cmd> lua LazyVim.pick() <cr>]]),
+        dashboard.button("n", " " .. " New file",        [[<cmd> ene <BAR> startinsert <cr>]]),
+        dashboard.button("r", " " .. " Recent files",    [[<cmd> lua LazyVim.pick("oldfiles") <cr>]]),
+        dashboard.button("g", " " .. " Find text",       [[<cmd> lua LazyVim.pick("live_grep") <cr>]]),
+        dashboard.button("c", " " .. " Config",          [[<cmd> lua LazyVim.pick.config_files() <cr>]]),
```

Comment:
```suggestion
        dashboard.button("f", " " .. " Find file",       LazyVim.pick() --[[@as string]]),
        dashboard.button("n", " " .. " New file",        [[<cmd> ene <BAR> startinsert <cr>]]),
        dashboard.button("r", " " .. " Recent files",    LazyVim.pick("oldfiles") --[[@as string]]),
        dashboard.button("g", " " .. " Find text",       LazyVim.pick("live_grep") --[[@as string]]),
        dashboard.button("c", " " .. " Config",          LazyVim.pick.config_files() --[[@as string]]),
```
I tested both and while `mini.starter` was working, in `alpha.nvim` it wasn't working without these changes.